steam/manifest.py

```diff
@@ -967,6 +970,13 @@ def __init__(
         """This app's review score."""
         self.review_percentage = int(common.get("review_percentage", 0))
         """This app's review percentage."""
+        self.metacritic_name = common.get("metacritic_name", None)
+        """This app's metacritic name."""
+        self.metacritic_score = int(common.get("metacritic_score")) if "metacritic_score" in common else None
+        """This app's metacritic score."""
+        self.metacritic_fullurl = common.get("metacritic_fullurl", None)
```

Comment:
Just url would suffice

Reply:
Is this not it? 

https://github.com/Gobot1234/steam.py/blob/b4a0bc7270691a84aadd684429e47dd7ed3c6e29/steam/types/manifest.py#L61

Reply:
ah, I should change the score to int

Reply:
Change it to VDFInt cause it is still technically a string

Reply:
Done